### DISCUSSION TITLE
Notifications are not working on iPhone. I need to enter the app in order to see them.

### DISCUSSION THREAD

**Discussion by abdalla-rko** [community/community#69694] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.133.1 (112099449)
Device: iPhone 13 Pro Max (iOS 17.0.3)
</details>

**Comment by esahmed21** [comment, marked as answer]
Hi @abdalla-rko,

Thank you for reaching out to us!

I'm sorry to hear that push notifications are not working with you!  Would you please make sure that you have push notifications types enabled and no custom working hours are set up? you can see that when you navigate to User Profile, then Settings, Notifications. Let me know what kind of settings you've so that we can help figuring out the problem

Thank you!
